upgrades/steps_27.go

```diff
@@ -77,17 +77,17 @@ func stateStepsFor27() []Step {
 			},
 		},
 		&upgradeStep{
-			description: "adds default value for default_space",
+			description: "replace space name in endpointBindingDoc bindings with an space ID",
 			targets:     []Target{DatabaseMaster},
 			run: func(context Context) error {
-				return context.State().AddDefaultSpaceSetting()
+				return context.State().ReplaceSpaceNameWithIDEndpointBindings()
 			},
 		},
 		&upgradeStep{
-			description: "replace space name in endpointBindingDoc bindings with an space ID",
+			description: `remove model config for default-space if set to "_default"`,
```

Comment:
nit: Wouldn't the description be more along the line of remove model config + set default/initialize to ""?